src/Swarm/TUI/Controller.hs

```diff
@@ -413,7 +421,7 @@ handleWorldEvent s (VtyEvent (V.EvKey (V.KChar 'c') [])) = do
 handleWorldEvent s (VtyEvent (V.EvKey (V.KChar 'p') [])) = do
   curTime <- liftIO $ getTime Monotonic
   continue $ s
-      & gameState . paused %~ not
+      & gameState . runStatus .~ (if s ^. gameState . runStatus == Running then ManualPause else Running)
```

Comment:
I think this could be more naturally written as `gameState . runStatus %~ (\stat -> if stat == Running then ManualPause else Running)`.  Well, at least it seems more natural to me. =)

Reply:
Oh indeed, thanks!